cmd/nvidia-cdi-hook/chmod/chmod.go

```diff
@@ -58,11 +59,11 @@ func (m command) build() *cli.Command {
 	c := cli.Command{
 		Name:  "chmod",
 		Usage: "Set the permissions of folders in the container by running chmod. The container root is prefixed to the specified paths.",
-		Before: func(c *cli.Context) error {
-			return validateFlags(c, &cfg)
+		Before: func(ctx context.Context, cmd *cli.Command) (context.Context, error) {
+			return ctx, validateFlags(cmd, &cfg)
```

Comment:
Let's update `validateFlags` and `m.run` to both have the signature: arguments:
```
(context.Context, *cli.Command, *config) (context.Context, error)
```
In this way they match the `Before` and `After` functors with the exception that they also accept the options.

Reply:
Let's rather leave them as is from the context of this change.